content/chainguard/chainguard-images/videos/kubectl_cdebug.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Debugging Distroless Images with Kubectl Debug and CDebug"
+linktitle: "Debugging with Kubectl and CDebug"
+lead: ""
+description: "This video explains how to use the Kubectl and CDebug tools to investigate
```

Comment:
I feel like I mostly see cdebug all lower case. Kubectl seems to come up both ways.

Reply:
Yeah, but it's CDebug in the title of the video which I can't change :/

Reply:
I've kept it as CDebug in titles and changed in body text. Not sure if that's more confusing 😅